plover/command/set_config.py

```diff
@@ -0,0 +1,41 @@
+import ast
+
+COMMAND_DELIMITER = ','
```

Comment:
I think I'd still prefer `:` as a command delim. I agree that `,` is pretty readable but we've been doing params in most plugins with colon. For the sake of consistency and setting an example from within Plover's codebase

Reply:
What about a colon to separate option:value, and commas to separate entire config settings? That allows for the syntax:

```{PLOVER:SET_CONFIG:option:value}```
```{PLOVER:SET_CONFIG:option1:value1,option2:value2:...}```

Which makes it easy to grab the command-line part, slap on a couple braces, and parse the whole thing as a dict literal. You could set any number of config options with one stroke that way. That feels too good to pass up...